### ISSUE TITLE
Finding reconstructed gene by cell from different models

### ISSUE THREAD

**Issue by Connorr0** [cistrome/MIRA#12]
I was hoping to look at the error in the final model on a per cell basis. I was hoping to get the reconstructed gene x cell matrices, or some measurement of loss for each cell. Is this information available in one of the objects/files in the pipeline?

To be clear, I am not thinking about just the topic values coding layer, I was hoping to get the output after the best model attempts to decode the compressed encoding.

**Comment by AllenWLynch** [comment]
Hi Connor,

Thanks for checking out MIRA! Just to clarify, from a trained topic model, you want to find the reconstruction loss on a per cell basis?

Allen
________________________________
From: Connor Finkbeiner ***@***.***>
Sent: Friday, October 14, 2022 6:44 PM
To: cistrome/MIRA ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [cistrome/MIRA] Finding reconstructed gene by cell from different models (Issue #12)


I was hoping to look at the error in the final model on a per cell basis. I was hoping to get the reconstructed gene x cell matrices, or some measurement of loss for each cell. Is this information available in one of the objects/files in the pipeline?

To be clear, I am not thinking about just the topic values coding layer, I was hoping to get the output after the best model attempts to decode the compressed encoding.

**Comment by Connorr0** [comment]
Yes, I wanted to look across cell types to see if the encoding was worse for any of my cell types.

**Comment by AllenWLynch** [comment]
Hi Connor,

Unfortunately, the current API does not support seeing per cell losses. I am working on a substantial update and will consider adding this feature.

Allen
________________________________

**Comment by Connorr0** [comment]
Thanks for the response!

**Comment by Connorr0** [comment]
Also, if I wanted to get an estimate of the reconstruction accuracy across cells would anndata[counts] vs model.impute() be meaningful (once I column normalized counts)?
